src/main/java/com/unosquare/cvgenerator/configuration/MapperConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.unosquare.cvgenerator.configuration;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MapperConfig {
+
+    @Bean
+    public ModelMapper setModelMapper() {
```

Comment:
Suggestion: for bean setup, naming convention is usually just the name of the bean:
```suggestion
    public ModelMapper modelMapper() {
```